src/App.js

```diff
@@ -1,53 +1,16 @@
 import React from 'react';
 import { Container } from 'react-bootstrap';
-import { NavLink, Route, Routes } from "react-router-dom";
 
-import MainPage from "./pages/MainPage/MainPage";
-import Basket from "./pages/Basket";
-import NotFound from "./pages/Not found/Not found";
-import { ROOT_TO_MAINPAGE,ROOT_TO_BASKET } from './routes'
+import MainPage from './pages/MainPage';
 
 import './App.css';
-
 function App() {
   return (
-    <div className="app">
+    <div className="App">
```

Comment:
better add `className` from lower case. 
And u don't have styles for class `.App`, but have for `.app` in file `App.css`